src/app/components/feed/jh-feed.html

```diff
@@ -39,7 +39,11 @@
     </div>
   </div>
   <figcaption>
-    <span class="caption">{{ vm.feed.display }}</span>
+    <span class="caption" ng-dblclick="vm.isEditable()" ng-hide="vm.nameIsEditable">{{vm.feed.display}}</span>
```

Comment:
on double click name becomes editable span disappears and the div appears, on clicking OK span again appears.